x-pack/test/security_solution_endpoint_api_int/apis/resolver/tree.ts

```diff
@@ -4,31 +4,22 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import expect from '@kbn/expect';
+import { Schema } from '../../../../plugins/security_solution/server/endpoint/routes/resolver/tree/utils';
```

Comment:
These changes are misleading. What I did was rename the old `tree.ts` file to `tree_entity_id.ts` which has the tests for the old api.

This file is brand new so I would ignore the deletions or just look at the raw source code when reviewing this. The tests in this file are for the new tree api route.